app/routes.py

```diff
@@ -1,2 +1,151 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.planet import Planet
+from app import db
 
+planets_bp = Blueprint("planets_bp", __name__, url_prefix = "/planets")
+
+# HELPER FUNCTION
+def validate_id(class_obj, id):
+    try:
+        object_id = int(id)
+    except:
+        abort(make_response({"message":f"{class_obj} {id} is invalid"}, 400))
+    
+    query_result = class_obj.query.get(object_id)
+    if not query_result:
+        abort(make_response({"message":f"{class_obj} {id} is not found"}, 404))
+
+    return query_result
+
+# CREATE RESOURCE
+@planets_bp.route("", methods = ["POST"])
+def create_planet():
+    request_body = request.get_json()
```

Comment:
Doing some validation on the request body here would be nice, just to check if it has the required fields.